components/Spaces/StudySpace/User.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Paper, Box, Typography, Avatar } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+
+import PersonIcon from "@material-ui/icons/Person";
```

Comment:
these are all external packages. Just group them imo

Reply:
![image](https://user-images.githubusercontent.com/60107091/121446613-e707f080-c961-11eb-902c-90d334cb28b5.png)
I can only group useTheme into the core package

Reply:
No I mean remove line 2 and line 5 lol.